include/net/nrf_cloud.h

```diff
@@ -523,6 +524,10 @@ struct nrf_cloud_init_param {
 	 * Only used if CONFIG_NRF_CLOUD_FOTA_FULL_MODEM_UPDATE is enabled.
 	 */
 	struct dfu_target_fmfu_fdev *fmfu_dev_inf;
+	/** Optional hooks to override memory management functions.
+	 *  If set, @ref nrf_cloud_os_mem_hooks_init will be called by @ref nrf_cloud_init.
```

Comment:
Note, that links to functions do not work as stated in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#references

Reply:
ok, I'll remove "ref"

Reply:
> ok, I'll remove "ref"

You don't necessarily need to remove hem, but just letting you know it would not render as a working link.

Reply:
ok